dagshub/data_engine/model/datasource.py

```diff
@@ -595,10 +595,38 @@ def _upload_metadata(self, metadata_entries: List[DatapointMetadataUpdateEntry])
         # Update the status from dagshub, so we get back the new metadata columns
         self.source.get_from_dagshub()
 
-    def _delete_datapoint(self, id):
-        self.source.client.delete_datapoint(self, DatapointDeleteEntry(datapointId=id))
+    def delete_datapoints(self, datapoints: List[Datapoint], force: bool = False):
+        """
+        Delete these datapoints.
+
+        - These datapoint will no longer show up in queries.
+        - Does not delete the datapoint's file, only removing the data from the datasource.
+        - You can still query these datapoints and associated metadata with \
+        versioned queries whose time is before deletion time.
+        - You can re-add these datapoints to the datasource by uploading new metadata to it with, for example, \
+        :func:`Datasource.metadata_context <dagshub.data_engine.model.datasource.Datasource.metadata_context>`. \
+        This will create a new datapoint with new id and new metadata records.
+        - Datasource scanning will *not* add these datapoints back.
+
+        Args:
+            datapoints: list of datapoints objects to delete
+            force: Skip the confirmation prompt
+        """
+        prompt = (
+            f'You are about to delete the datapoints {[d.path for d in datapoints]}."\n'
+            f"This will remove the datapoint and metadata from unversioned queries, "
+            f"but won't delete the underlying file."
```

Comment:
Print out the paths on a separate line with a \t added.
Something like (didn't test it, just typing it out here)
```suggestion
            f"You are about to delete the following datapoint(s):\n"
            f"{'\n'.join(['\t'+d.path for d in datapoints])}\n"
            f"This will remove the datapoint(s) and metadata from unversioned queries, "
            f"but won't delete the underlying file(s)."
```